product-info-service/handlers/getAllProducts.ts

```diff
@@ -0,0 +1,14 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import * as products from '../staticData/productsList.json';
+import 'source-map-support/register';
+import { headers } from "../constants";
+
+export const getAllProducts: APIGatewayProxyHandler = async (_event, _context) => {
+  return {
+    statusCode: 200,
+    body: JSON.stringify({
+      products
+    }),
+    headers
+  };
+}
```

Comment:
It's better to wrap code in handlers in try and catch blocks because could be some exceptions in "real world" handlers. For example, connection with database wasn't successful.  

Anyway, it's required for task3:

> +1 - Main error scenarious are handled by API ("Product not found" error, try catch blocks are used in lambda handlers).